src/mavsdk/plugins/info/info_impl.cpp

```diff
@@ -46,17 +48,16 @@ void InfoImpl::deinit()
 
 void InfoImpl::enable()
 {
-    // We can't rely on System to request the autopilot_version,
-    // so we do it here, anyway.
-    _system_impl->send_autopilot_version_request();
-    _system_impl->send_flight_information_request();
+    // We're going to periodically ask for the flight information
+    if (_system_impl->autopilot() != SystemImpl::Autopilot::ArduPilot) {
+        _system_impl->send_flight_information_request();
+
+        _system_impl->add_call_every(
+            [this]() { request_flight_information(); }, 1.0f, &_flight_info_call_every_cookie);
+    }
 
     // We're going to retry until we have the version.
     _system_impl->add_call_every([this]() { request_version_again(); }, 1.0f, &_call_every_cookie);
-
-    // We're going to periodically ask for the flight information
-    _system_impl->add_call_every(
-        [this]() { request_flight_information(); }, 1.0f, &_flight_info_call_every_cookie);
```

Comment:
Shouldn't this one stay for PX4?

Reply:
This one wasn't removed, it was moved into the if statement

Reply:
Oh, right :man_facepalming: